code/tests/test_vector_stores.py

```diff
@@ -99,16 +99,12 @@ def test_deleted_expired_data(input_db, crawl_2, request):
 
     db = request.getfixturevalue(input_db)
     # Delete expired objects
-    res = db.search_by_vector(db.dummy_vector, k=10, filter_={"last_seen_at": {"$lt": 1}})
-    assert res, "Expected expired objects in the database"
-    assert len(res) == 1, "Expected 1 expired object in the database"
-
```

Comment:
These filters are specific to Pinecone and Chroma.
I've removed them instead of adding some conditionals in-line since the lines below should cover this test.